frontend/public/components/dashboard/dashboards-page/dashboards.tsx

```diff
@@ -63,25 +71,23 @@ const getTabs = (flags: FlagsObject): Page[] => [
   ...getPluginTabPages(flags),
 ];
 
-const DashboardsPage_: React.FC<DashboardsPageProps> = ({
-  match,
-  kindsInFlight,
-  k8sModels,
-  flags,
-}) => {
-  const title = 'Overview';
-  return kindsInFlight && k8sModels.size === 0 ? (
-    <LoadingBox />
-  ) : (
-    <>
-      <Helmet>
-        <title>{title}</title>
-      </Helmet>
-      <PageHeading title={title} detail={true} />
-      <HorizontalNav match={match} pages={getTabs(flags)} noStatusBox />
-    </>
-  );
-};
+const DashboardsPage_ = withFallback<DashboardsPageProps>(
```

Comment:
I feel like it would be better to have the error boundaries around the individual cards. Then if there's an error in a specific card, the rest of the dashboard still loads.

Reply:
Ok, I've added error boundaries for DashboardCard component

![Screenshot from 2020-02-28 13-45-22](https://user-images.githubusercontent.com/2078045/75551984-9fdc4180-5a35-11ea-81f1-a2e46a7b0894.png)